main.js

```diff
@@ -0,0 +1,220 @@
+// Set current date
+const date = new Date();
+const options = { weekday: 'long'};
+const formattedDate = new Intl.DateTimeFormat('en-US', options).format(date);
+
+
+const formattedDates = [];
+
+for (let i = 0; i < 5; i++) {
+    const nextDay = new Date(date);
+    nextDay.setDate(date.getDate() + i + 1);
+    const formattedDate = new Intl.DateTimeFormat('en-US', options).format(nextDay);
+    formattedDates.push(formattedDate); 
+}
+
+// Get current weather
+
+var currentWeather = [];
+
+var weatherForecast = [];
+
+document.querySelector('.search').addEventListener('click', function () {
+  var searchLocation = document.querySelector('#search-query').value;
+  var words = searchLocation.split(' ');
+  var city = words.join('/');
+
+  document.querySelector('#search-query').value = '';
+  currentWeather = [];
+  weatherForecast = [];
+
+  fetchLocation(city);
+  fetchPlace(city);
+});
+
+var fetchLocation = function(city) {
+  const apiKey = '0e79198cac257d0139d13e84e8617759'
+  const url = `http://api.openweathermap.org/geo/1.0/direct?q=${city}&appid=` + apiKey;
+  fetch(url, {
+    method: 'GET',
+    dataType: 'json'
+  })
+  .then(data => data.json())
+  .then(data => fetchWeather(data));
+};
+
+var fetchWeather = function(data) {
+  const apiKey = '0e79198cac257d0139d13e84e8617759'
```

Comment:
api key is the same, can be defined once at the top of the file or passed a param